src/entity/SignedOrderEntity.ts

```diff
@@ -1,65 +1,99 @@
+
+    @Column({ name: 'taker_asset_data', type: 'varchar' })
+    public takerAssetData?: string;
+
+    @Column({ name: 'exchange_address', type: 'varchar' })
+    public exchangeAddress?: string;
+
+    @Column({ name: 'fee_recipient_address', type: 'varchar' })
+    public feeRecipientAddress?: string;
+
+    @Column({ name: 'expiration_time_seconds', type: 'int' })
+    public expirationTimeSeconds?: number;
+
+    @Column({ name: 'maker_fee', type: 'varchar' })
+    public makerFee?: string;
+
+    @Column({ name: 'taker_fee', type: 'varchar' })
+    public takerFee?: string;
+
+    @Column({ name: 'maker_asset_amount', type: 'varchar' })
+    public makerAssetAmount?: string;
+
+    @Column({ name: 'taker_asset_amount', type: 'varchar' })
+    public takerAssetAmount?: string;
+
+    @Column({ name: 'salt', type: 'varchar' })
+    public salt?: string;
+
+    @Column({ name: 'signature', type: 'varchar' })
+    public signature?: string;
+
+    @Column({ name: 'remaining_fillable_taker_asset_amount', type: 'varchar' })
+    public remainingFillableTakerAssetAmount?: string;
+
+    @Column({ name: 'maker_fee_asset_data', type: 'varchar' })
+    public makerFeeAssetData?: string;
+
+    @Column({ name: 'taker_fee_asset_data', type: 'varchar' })
+    public takerFeeAssetData?: string;
+    constructor(
+        opts: {
+            hash?: string;
+            senderAddress?: string;
+            makerAddress?: string;
+            takerAddress?: string;
+            makerAssetData?: string;
+            takerAssetData?: string;
+            exchangeAddress?: string;
+            feeRecipientAddress?: string;
+            expirationTimeSeconds?: number;
+            makerFee?: string;
+            takerFee?: string;
+            makerFeeAssetData?: string;
+            takerFeeAssetData?: string;
+            makerAssetAmount?: string;
+            takerAssetAmount?: string;
+            salt?: string;
+            signature?: string;
+            remainingFillableTakerAssetAmount?: string;
+        } = {},
+    ) {
+        this.hash = opts.hash;
+        this.senderAddress = opts.senderAddress;
+        this.makerAddress = opts.makerAddress;
+        this.takerAddress = opts.takerAddress;
+        this.makerAssetData = opts.makerAssetData;
+        this.takerAssetData = opts.takerAssetData;
+        this.exchangeAddress = opts.exchangeAddress;
+        this.feeRecipientAddress = opts.feeRecipientAddress;
+        this.expirationTimeSeconds = opts.expirationTimeSeconds;
+        this.makerFee = opts.makerFee;
+        this.takerFee = opts.takerFee;
+        this.makerFeeAssetData = opts.makerFeeAssetData;
+        this.takerFeeAssetData = opts.takerFeeAssetData;
+        this.makerAssetAmount = opts.makerAssetAmount;
+        this.takerAssetAmount = opts.takerAssetAmount;
+        this.salt = opts.salt;
+        this.signature = opts.signature;
+        this.remainingFillableTakerAssetAmount = opts.remainingFillableTakerAssetAmount;
```

Comment:
I wonder if there is any way we don't need to make changes in 3 different places in order to add a new property

Reply:
I can't think of a really clean way. FWIW this object existed before I just moved it here.